test/git_tools.jl

```diff
@@ -107,10 +107,7 @@ function tree_hash(::Type{HashType}, root::AbstractString) where HashType
         filepath = abspath(root, f)
         mode = gitmode(filepath)
         if mode == mode_dir
-            # If this directory contains no files, then skip it
-            contains_files(filepath) || continue
-
-            # Otherwise, hash it up!
+            # git skips emtpy directories, this doesn't
```

Comment:
Isn't this going to break all the hashes that are out there right now with empty directories?

Reply:
This is just a test utility, it doesn't affect anything besides the tests.

Reply:
Ah, I missed the `test/` prefix. :P